Add clonable option and attribute (#31849)
mdn/content: files/en-us/web/api/shadowroot/clonable/index.md

---
title: "ShadowRoot: clonable property"
short-title: clonable
slug: Web/API/ShadowRoot/clonable
page-type: web-api-instance-property
browser-compat: api.ShadowRoot.clonable
---

{{APIRef("Shadow DOM")}}

> The **`clonable`** read-only property of the {{domxref("ShadowRoot")}} interface returns `true` if the shadow root is clonable, and `false` otherwise. It's always returns `true` for shadow roots created via declarative shadow DOM or can be set via {{domxref("Element.attachShadow()")}} `clonable` option.

Reviewer: - A nit with "It's"
- How about splitting the last sentence for clarity?
```suggestion
The **`clonable`** read-only property of the {{domxref("ShadowRoot")}} interface returns `true` if the shadow root is clonable, and `false` otherwise. It always returns `true` for shadow roots created via declarative shadow DOM. It can be set to `true` using the `clonable` option of the {{domxref("Element.attachShadow()")}} method.
```
